Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/azure/serverless/spark/models/AnalyticsActivity.java

```diff
@@ -145,27 +179,65 @@ public String submitter() {
     }
 
     /**
-     * Get the state value.
+     * Get state in which the activity is in from the perspective of the scheduler. These states are common across different activity types. Kept here for backward compatibility. Soon to be deprecated. Possible values include: 'Any', 'Submitted', 'Preparing', 'Queued', 'Scheduled', 'Finalizing', 'Ended'.
      *
      * @return the state value
      */
-    public ActivityState state() {
+    public SchedulerState state() {
         return this.state;
     }
 
     /**
-     * Set the state value.
+     * Set state in which the activity is in from the perspective of the scheduler. These states are common across different activity types. Kept here for backward compatibility. Soon to be deprecated. Possible values include: 'Any', 'Submitted', 'Preparing', 'Queued', 'Scheduled', 'Finalizing', 'Ended'.
      *
      * @param state the state value to set
      * @return the AnalyticsActivity object itself.
      */
-    public AnalyticsActivity withState(ActivityState state) {
+    public AnalyticsActivity withState(SchedulerState state) {
         this.state = state;
         return this;
     }
 
     /**
-     * Get the submitTime value.
+     * Get state in which the activity is in from the perspective of the scheduler. These states are common across different activity types. Possible values include: 'Any', 'Submitted', 'Preparing', 'Queued', 'Scheduled', 'Finalizing', 'Ended'.
+     *
+     * @return the schedulerState value
+     */
+    public SchedulerState schedulerState() {
+        return this.schedulerState;
+    }
+
+    /**
+     * Set state in which the activity is in from the perspective of the scheduler. These states are common across different activity types. Possible values include: 'Any', 'Submitted', 'Preparing', 'Queued', 'Scheduled', 'Finalizing', 'Ended'.
+     *
+     * @param schedulerState the schedulerState value to set
+     * @return the AnalyticsActivity object itself.
+     */
+    public AnalyticsActivity withSchedulerState(SchedulerState schedulerState) {
+        this.schedulerState = schedulerState;
+        return this;
+    }
+
+    /**
+     * Get state in which the activity is in from the perspective of the activity plugin. The value set for this state differs for each activity type.
+     *
+     * @return the activityState value
+     */
+    public String activityState() {
+        return this.activityState;
+    }
+
+    /**
+     * Get result of the activity. Possible values include: 'None', 'Succeeded', 'Cancelled', 'Failed'.
+     *
+     * @return the result value
+     */
+    public ActivityResult result() {
```

Comment:
The property is called just result in the response, we should keep it the same for consistency instead of ActivityResult